src/ethers/contracts/sylvester/utils.ts

```diff
@@ -24,13 +27,13 @@ export const createSylvesterV0LendThunk = (
   options?: any
 ): Promise<ContractTransaction> => {
   const args = prepareBatch({
-    nftStandard,
```

Comment:
can we not re-order these to keep the mental model of which arguments go first into the smart contract wallet intact?

So everything in prepareBatch should not be re-ordered across this whole file.